CMakeLists.txt

```diff
@@ -123,7 +123,10 @@ find_package(HEPMC)
 # Check the compiler and set the compile and link flags
 check_compiler()
 
-#set(CMAKE_CXX_FLAGS -std=c++14)
+if(NOT DEFINED ENV{NO_WERROR})
```

Comment:
Switching -Wall and -Werror on by default causes many problems on different platforms. I propose to have it optional and off by default